index.js

```diff
@@ -14,4 +14,128 @@ cloSerX.addEventListener('click', () => {
 navListItems.forEach((item) => item.addEventListener('click', () => {
   mariMarzi.classList.add('hide');
   mariMarzi.classList.remove('mobile-menu');
-}));
\ No newline at end of file
+}));
+function createPopupWindow(project) {
```

Comment:
- [ OPTIONAL] Whenever possible, I suggest that you should always use array functions. They are **concise & easier to read**   🤝